### DISCUSSION TITLE
How large should my study be?

### DISCUSSION THREAD

**Discussion by rdstern** [IDEMSInternational/R-Instat#8023] [Ideas]
@fran2or and  @IssoufKon   I have found a really nice resource - I think.  It is called [Sample Size Calculation with R](https://med.und.edu/daccota/_files/pdfs/berdc_resource_pdfs/sample_size_r_module.pdf).  I suggest it could lead to a good dialogue in the Experiments menu,  this would be in the planning or design section.

It is reasonably comprehensive and lists the R packages for the different designs.  And it includes small exercises that we could make use of - perhaps in STACK.

It could also illustrate the topic of "When you have finished this, then will you know enough about sample-size calculation for your own problem?  What is discussed here is important, but is it sufficient for you?  This could even be phrased as a play/video on getting statistical advice.  You are directed to a statistician who is working on this very topic.  He discusses this and asks you to go through it.

What doesn't it do.  You would know the answers for many designs, but then have to link this to the particular design you propose to use.  Assumes just one objective, i.e. one variable and that your analysis will be a significance test.  And a single study.

Often the sample size is dictated by your funds and resources for the work.  Then these results could still be useful perhaps.  You include the sample size and see what difference you can detect.

Stepping back even further we look at the size and other aspects of plant breeding studies.  I wonder if we can get, from on-station work, to on-farm, to the size of the recent Fuma Gaskiya work?

**Comment by fran2or** [comment]
@IssoufKon

**Reply by IssoufKon** [reply]
Hi Roger, I find this resource interesting, I will read it to understand more. Maybe we can discuss it in our meeting together.
